docs/sources/panels/transformations/types-options.md

```diff
@@ -0,0 +1,285 @@
++++
+title = "Transformation Types and Options"
+type = "docs"
+[menu.docs]
+identifier = "transformation_types_and_options"
+parent = "transformations"
+weight = 300
++++
+
+# Transformation types and options
+
+Grafana comes with the following transformations:
+  
+  - [Reduce](#reduce)
+  - [Merge](#merge)
+  - [Filter by name](#filter-by-name)
+  - [Filter data by query](#filter-data-by-query)
+  - [Organize fields](#organize-fields)
+  - [Join by field (outer join)](#join-by-field-outer-join)
+  - [Add field from calculation](#add-field-from-calculation)
+  - [Labels to fields](#labels-to-fields)
+  - [Group By](#group-by)
```

Comment:
```suggestion
  - [Group by](#group-by)
```

Reply:
The UI says "Group by." Reconcile throughout and in the What's new in 7.2 topic.

Reply:
Done